pytorch_lightning/callbacks/progress/tqdm_progress.py

```diff
@@ -207,12 +208,10 @@ def init_test_tqdm(self) -> Tqdm:
         return bar
 
     def on_sanity_check_start(self, trainer, pl_module):
-        super().on_sanity_check_start(trainer, pl_module)
```

Comment:
why is it necessary to remove the `super` calls?

Reply:
The parent doesn't override these. It's not necessary, I can put them back if you want.